solr/solr-ref-guide/src/jwt-authentication-plugin.adoc

```diff
@@ -21,6 +21,11 @@ Solr can support https://en.wikipedia.org/wiki/JSON_Web_Token[JSON Web Token] (J
 This allows Solr to assert that a user is already authenticated with an external https://en.wikipedia.org/wiki/Identity_provider[Identity Provider] by validating that the JWT formatted https://en.wikipedia.org/wiki/Access_token[access token] is digitally signed by the Identity Provider.
 The typical use case is to integrate Solr with an https://en.wikipedia.org/wiki/OpenID_Connect[OpenID Connect] enabled IdP.
 
+== Contrib package
+
+This is provided via a Solr Contrib package that needs to be added to the classpah before use. Since this is a node-level
+plugin it must go in `sharedLib`, see <<configuring-solr-xml.adoc#,Configuring solr.xml>> for details.
```

Comment:
I tested this manually with 
```bash
bin/solr start -c -Dsolr.sharedLib=/Users/janhoy/git/solr/solr/packaging/build/dev/dist/,/Users/janhoy/git/solr/solr/packaging/build/dev/contrib/jwt-auth/lib/
```
and it then worked.

I first tried adding the two jars in sharedLib, but sharedLib only accepts folders. Would be useful if it also accepted a jar or a wildcard, like the `<lib>` in solrconfig.